### PR TITLE
gtk-doc 1.26

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
==> /usr/local/Cellar/gtk-doc/1.26/bin/gtkdoc-scan --module=test
Traceback (most recent call last):
  File "/usr/local/Cellar/gtk-doc/1.26/bin/gtkdoc-scan", line 29, in <module>
    from gtkdoc import common, config, scan
  File "/usr/local/Cellar/gtk-doc/1.26/share/gtk-doc/python/gtkdoc/scan.py", line 37, in <module>
    from six import iteritems, iterkeys
ImportError: No module named six
```

**Comment by ilovezfs** [comment]
@tschoonj I think it needs something like this:
```
diff --git a/Formula/gtk-doc.rb b/Formula/gtk-doc.rb
index 938f70f7fc..4d1a9a2c06 100644
--- a/Formula/gtk-doc.rb
+++ b/Formula/gtk-doc.rb
@@ -22,15 +22,27 @@ class GtkDoc < Formula
   depends_on "libxml2"
   depends_on :perl => "5.18" if MacOS.version <= :mavericks
 
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/b3/b2/238e2590826bfdd113244a40d9d3eb26918bd798fc187e2360a8367068db/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
   def install
     ENV.append_path "PYTHONPATH", "#{Formula["libxml2"].opt_lib}/python2.7/site-packages"
 
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    resource("six").stage do
+      system "python", *Language::Python.setup_install_args(libexec/"vendor")
+    end
+
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--with-xml-catalog=#{etc}/xml/catalog"
     system "make"
     system "make", "install"
+
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
   test do
```

File: Formula/gtk-doc.rb [hunk 1]
```diff
@@ -13,24 +13,34 @@ class GtkDoc < Formula
...
 
   depends_on "pkg-config" => :build
-  depends_on "gnome-doc-utils" => :build
   depends_on "itstool" => :build
   depends_on "gettext"
-  depends_on "glib"
   depends_on "docbook"
   depends_on "docbook-xsl"
   depends_on "libxml2"
   depends_on :perl => "5.18" if MacOS.version <= :mavericks
```
Comment (ilovezfs) [inline Formula/gtk-doc.rb]: Is this still needed given the Python rewrite?

Comment (tschoonj) [inline Formula/gtk-doc.rb]: what are you referring to exactly?

Comment (ilovezfs) [inline Formula/gtk-doc.rb]:
```
depends_on :perl => "5.18" if MacOS.version <= :mavericks
```

Comment (tschoonj) [inline Formula/gtk-doc.rb]: I am actually not sure why there is a Perl dependency there at all. There doesn't appear to be any Perl code in the repository

Comment (ilovezfs) [inline Formula/gtk-doc.rb]: I think the prior version to this one used some Perl in places where this version now uses Python.

Comment (tschoonj) [inline Formula/gtk-doc.rb]: Ok, I removed the Perl line.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
:white_check_mark:

**Pull request closed without merging**